CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Under the hood
 - Adding `execution_project` to `target` object ([#66](https://github.com/dbt-labs/dbt-bigquery/issues/66))
+- Fix test related to preventing coercion of boolean values (True, False) to numeric values (0, 1) in query results ([#93](https://github.com/dbt-labs/dbt-bigquery/issues/93))
```

Comment:
I think we need to change the section this is in @McKnight-42. Could you open another PR that creates a new section above `dbt-bigquery 1.1.0 (Release TBD)` and then change the release date for `dbt-bigquery 1.0.0` to be December 3, 2021? I think we missed updating this when we did the release

Reply:
Do we still want it in the under the hood section of the tbd or placed in fixes?

Reply:
Do we still want it in the under the hood section of the tbd or placed in fixes?

Reply:
Let's start a new `Fixes` section under `1.1.0` and put this under there

Reply:
done, changes made in a new pr #95